src/plone/restapi/tests/test_documentation.py

```diff
@@ -1345,7 +1345,7 @@ def test_querystringsearch_post(self):
 
 class TestDocumentationMessageTranslations(TestDocumentationBase):
 
-    layer = layer = PLONE_RESTAPI_DX_FUNCTIONAL_TESTING
```

Comment:
Good catch. How the hell did that end up here? :)

Reply:
:smile: